enterprise/cmd/frontend/internal/auth/githubappauth/resolver_test.go

```diff
@@ -136,41 +140,64 @@ func TestResolver_GitHubApps(t *testing.T) {
 	schema, err := graphqlbackend.NewSchema(db, gitserver.NewClient(), nil, graphqlbackend.OptionalResolver{GitHubAppsResolver: NewResolver(logger, db)})
 	require.NoError(t, err)
 
-	graphqlbackend.RunTests(t, []*graphqlbackend.Test{{
-		Schema:  schema,
-		Context: adminCtx,
-		Query: `
-			query GitHubApps() {
-				gitHubApps {
-					nodes {
-						id
+	graphqlbackend.RunTests(t, []*graphqlbackend.Test{
+		{
+			Schema:  schema,
+			Context: adminCtx,
+			Query: `
+				query GitHubApps() {
+					gitHubApps {
+						nodes {
+							id
+						}
 					}
+				}`,
+			ExpectedResult: fmt.Sprintf(`{
+				"gitHubApps": {
+					"nodes": [
+						{"id":"%s"},
+						{"id":"%s"}
+					]
 				}
-			}`,
-		ExpectedResult: fmt.Sprintf(`{
-			"gitHubApps": {
-				"nodes": [
-					{"id":"%s"}
-				]
-			}
-		}`, graphqlID),
-	}, {
-		Schema:  schema,
-		Context: userCtx,
-		Query: `
-			query GitHubApps() {
-				gitHubApps {
-					nodes {
-						id
+			}`, graphqlID, graphqlID2),
+		},
+		{
+			Schema:  schema,
+			Context: adminCtx,
+			Query: `
+				query GitHubApps() {
+					gitHubApps(domain: "repos") {
```

Comment:
I think these tests might be failing at the moment:

```suggestion
					gitHubApps(domain: "REPOS") {
```